src/plugins/starter/starter-content.ts

```diff
@@ -0,0 +1,27 @@
+import { types, Instance } from "mobx-state-tree";
+import { ToolContentModel } from "../../models/tools/tool-types";
+import { kStarterToolID } from "./starter-types";
+
+export function defaultStarterContent(): StarterContentModelType {
+  return StarterContentModel.create();
+}
+
+
+export const StarterContentModel = ToolContentModel
+  .named("StarterTool")
+  .props({
+    type: types.optional(types.literal(kStarterToolID), kStarterToolID),
+    text: "Hello World",
```

Comment:
```suggestion
    text: "",
```
It may not matter in the long run, but it seems like "Hello World" should be the `defaultStarterContent()` rather than the core model default.